test.js

```diff
@@ -575,78 +575,53 @@ test('single put with overridden ttl set (custom ttlEncoding)', put(200, { ttl:
   ttlEncoding: bytewise
 })
 
-function testBatchPutWithDefaultTtl (t, db) {
-  db.batch([
-    { type: 'put', key: 'foo', value: 'foovalue' },
-    { type: 'put', key: 'bar', value: 'barvalue' }
-  ], function (err) {
-    t.ok(!err, 'no error')
-    setTimeout(function () {
-      db.get('foo', function (err, value) {
-        t.notOk(err, 'no error')
-        t.equal('foovalue', value)
-        db.get('bar', function (err, value) {
+function batch (timeout, opts) {
```

Comment:
@vweevers Using your technique here to return different test functions based on some configuration, based on a `timeout` value and `opts` to `db.batch()`.